ci/prow/boskos/create_projects.sh

```diff
@@ -16,18 +16,25 @@
 
 set -e
 
-readonly FIRST=${1:?"First argument is the first number of the new project(s)."}
```

Comment:
As discussed in person, move this change to a separate PR. It's a (good) enhancement, but unrelated to the permissions.

Reply:
Done in https://github.com/knative/test-infra/pull/1405